state/state.go

```diff
@@ -120,6 +120,7 @@ func (s *State) getCachedObject(addr thor.Address) (*cachedObject, error) {
 	if err != nil {
 		return nil, err
 	}
+	metricAccountCounter().AddWithLabel(1, map[string]string{"type": "read", "target": "trie"})
```

Comment:
`State.GetAccount` -> `stackedMap.Get` -> `state.getCachedObject` -> `state.LoadAccount` -> `trie.Get`

This might not be reflecting the real calling path, but just for demonstrating the path from state to the trie package. I would suggest consider in the low-level packages other than this one.

Reply:
The problem of this approach is that we miss the concept of object, so at  the `trie` level we do not know whether is an `account` or something else. That method is actually used also by the `chain` package.

Reply:
Let's solve this offline.